app/src/renderer/views/Inbox/MainContent/Conversation/Item/Reply.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Item } from "../../../../../../types";
+import { useTranslation } from "react-i18next";
+import "../Item.css";
+
+interface ReplyProps {
+  item: Item;
+}
+
+function Reply({ item }: ReplyProps) {
+  const { t } = useTranslation("MainContent");
+  const isEncrypted = !item.plaintext;
+  const messageContent = item.plaintext || "";
+
+  return (
+    <div
+      className="flex items-start mb-4 justify-end"
+      data-testid={`item-${item.uuid}`}
+    >
+      <div>
+        <div className="flex items-center justify-start mb-1">
+          <span className="author reply-author">You</span>
```

Comment:
In a follow-up: this needs to be localized. Also replies might be sent by another journalist, not just the current user.